modules/swagger-codegen/src/main/resources/flaskConnexion/util.mustache

```diff
@@ -44,7 +44,12 @@ def _deserialize_primitive(data, klass):
     try:
         value = klass(data)
     except UnicodeEncodeError:
-        value = six.u(data)
+        if isinstance(data, unicode):
```

Comment:
I would do this before the `try` block.

Reply:
@yucer did you have a chance to review the feedback?

Reply:
sorry I did not  see the feedback before. should I still do something ?